packages/babel-compat-data/data/plugin-bugfixes.json

```diff
@@ -121,5 +121,21 @@
     "ios": "13",
     "samsung": "3.4",
     "electron": "0.21"
+  },
+  "proposal-optional-chaining": {
+    "chrome": "80",
+    "opera": "67",
+    "edge": "80",
+    "firefox": "74",
+    "safari": "13.1",
+    "node": "14",
+    "ios": "13.4",
+    "samsung": "13",
+    "electron": "8.0"
+  },
+  "bugfix/transform-v8-spread-parameters-after-optional-chaining": {
```

Comment:
WDYT `bugfix/v8-spread-parameters-after-optional-chaining`? I can't think of any term other `transform` that could follow `bugfix/`.

Reply:
Nit: `in`, not `after`?